examples/foundry/src/Counter.sol

```diff
@@ -17,4 +17,8 @@ contract Counter is SapphireDecryptor {
     function withdraw() external {
         payable(msg.sender).transfer(address(this).balance);
     }
+
+    function thisCompilesToPush0AndShouldDetectWrongEvmVersion() public pure returns (bool) {
+        return false; // This will compile to PUSH0 in new unsupported EVM versions
+    }
```

Comment:
nevermind :shrug: 

This deploys to Sapphire Testnet successfully
```
pragma solidity ^0.8.13;

contract A {
    function thisCompilesToPush0AndShouldDetectWrongEvmVersion() public pure returns (bool) {
        return false; // This will compile to PUSH0 in new unsupported EVM versions
    }
}
```